packages/app/src/systems/Bridge/components/BridgeTxItem/BridgeTxItem.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import { cssObj } from '@fuel-ui/css';
```

Comment:
this story is not looking good 
![image](https://github.com/FuelLabs/fuels-portal/assets/8636507/213abd61-38c9-40e8-850f-5e9233d468bb)